Autopilot/Inc/airspeed.hpp

```diff
@@ -84,6 +86,26 @@ class dummyairspeed: public airspeed{
 
 };
 
+class MPXV7002DP : public airspeed {
+    public:
+        MPXV7002DP(const MPXV7002DP*) = delete;
+        static MPXV7002DP* GetInstance();
+        void GetResult(airspeedData_t &Data);
+        void Begin_Measuring();
+        void Init();
+    private:
+        MPXV7002DP();
+        static MPXV7002DP* s_Instance; //single instance of airspeed
+        //command to interface with MPXV7002
+        double readFromMPXV7002DP();
+        int offset = 0;
```

Comment:
If I understand the code correctly could this be renamed to something like "zeroVelocityOffset" or something?